.yamllint

```diff
@@ -15,3 +15,6 @@ rules:
   line-length: disable
   truthy: disable
   comments-indentation: disable
+  indentation:
```

Comment:
@sunkickr This should fix the precommit linting errors on `settings/testfiles/airflow_settings_export.yaml`